static/app/routes.tsx

```diff
@@ -48,6 +48,14 @@ const routeHook = (name: HookName): SentryRouteObject => {
   };
 };
 
+const routeObjectHook = (name: HookName): SentryRouteObject => {
+  const route = HookStore.get(name)?.[0] ?? {};
+  return {
+    ...route,
+    deprecatedRouteProps: true,
```

Comment:
Why aren't you able to just set this on the getsentry side?

Reply:
wait why do we even need this function, can we not just use `routeHook`?

Reply:
Right but can't we just make the settings route object a function so it still uses this same signature?